src/FileUploader/FileUploader.svelte

```diff
@@ -95,11 +95,11 @@
     kind="{kind}"
     on:change
     on:change="{({ target }) => {
-      files = [...target.files].map(({ name }) => name);
+      files = [...target.files];
```

Comment:
```suggestion
      files = target.files;
```

To align with the `FileList` type, `target.files` should not be spread.